### ISSUE TITLE
Arrays in Stores Transformed Upon Retrieval

### ISSUE THREAD

**Issue by daniel-klingensmith-jamf** [outofcoffee/imposter#601]
After storing JSON containing an array in a store, it is transformed upon retrieving:

`POST https://<lambda>.lambda-url.us-east-1.on.aws/system/store/test`
Input:
```
{
    "foo": [
        "bar"
    ]
}
```

`GET https://<lambda>.lambda-url.us-east-1.on.aws/system/store/test`
Output:
```
{
    "foo": {
        "list": [
            "bar"
        ],
        "empty": false
    }
}
```

Using 3.44.1, also found issue on 3.35.1.
Using imposter-awslambda.zip

**Comment by outofcoffee** [comment]
Hi @daniel-klingensmith-jamf, thank you for raising this. This should be fixed in [v4.0.2](https://github.com/outofcoffee/imposter/releases).

**Comment by daniel-klingensmith-jamf** [comment]
I've verified this works in 4.0.3. Thank you!!!
